src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/index.tsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react'
-import classNames from 'classnames'
 import codeBlockPrimitives from '@hashicorp/react-code-block/mdx'
 import EnterpriseAlertBase from '@hashicorp/react-enterprise-alert'
 import { useCurrentProduct } from 'contexts'
 import {
   MdxOrderedList,
   MdxUnorderedList,
   MdxListItem,
-} from 'components/dev-dot-content/mdx-components/mdx-lists'
+  MdxTab,
+  MdxTabs,
+} from 'components/dev-dot-content/mdx-components'
```

Comment:
Both docs and tutorials MDX component files now import `MdxTabs` & `MdxTab` from `components/dev-dot-content/mdx-components`.